pom.xml

```diff
@@ -1600,6 +1600,19 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.6</id>
+      <properties>
+        <hadoop.version>2.6.0</hadoop.version>
```

Comment:
is something specific from hadoop 2.6 needed for this patch? ie encryption will only work when built against hadoop 2.6?